build.gradle

```diff
@@ -11,344 +11,333 @@
  *   https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_source_sets
  */
 
-import io.swagger.codegen.config.CodegenConfigurator
-import io.swagger.codegen.DefaultGenerator
-
 buildscript {
-    ext {
-        springBootVersion = '2.3.4.RELEASE'
-    }
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
-        classpath('io.swagger:swagger-codegen:2.4.11')
-        classpath('gradle.plugin.com.srcclr:gradle:3.1.0')
-    }
+  ext {
+    springBootVersion = '2.3.4.RELEASE'
```

Comment:
What motivated the indentation change? Ideally, it would be good to do that as a separate step, since now the whole file is different.